src/treeViewLemma.ts

```diff
@@ -105,10 +119,10 @@ export class Entry extends vscode.TreeItem {
         const command =
           item.level === 4
             ? {
-                command: "extension.openFile",
+                command: "vscode.open",
                 title: "Open File",
                 arguments: [
-                  `${Entry.rootPath}/data/eng/${language}/${item.fileName}${wordExtension}`,
+                  `file:///${Entry.rootPath}/${languagePath}${languageCode}/${language}/${item.fileName}${wordExtension}`,
```

Comment:
This isn't working for me. I was able to get this to work by using the workspace folder's URI and `Uri.joinPath`.

Reply:
hmm... Don't know why it works for me but not for you.   Can you post the code that works for you and I will modify and see if it also works for me.

Reply:
Here is the code that I used for the URL:
```ts
vscode.Uri.joinPath(
  vscode.workspace.workspaceFolders![0].uri, 
  `${languagePath}${languageCode}/${language}/${item.fileName}${wordExtension}`
)
```